docs/modules/ROOT/pages/how-tos/testing/integration/adding.adoc

```diff
@@ -40,3 +40,5 @@ When the new build is finished:
 .  On the *Details* page, you can see if the test succeeded for that component. Navigate to the other tabs for more details. 
 
 
+NOTE: Now you know how to add integration test, in case you want to make a custom one, please continue to next section: *Creating a custom integration test*
```

Comment:
Can you add an xref to the next section?

Reply:
Updated.